operators/src/pro/machine_learning/metadata_from_file.rs

```diff
@@ -0,0 +1,93 @@
+use std::path::Path;
+
+use geoengine_datatypes::{machine_learning::MlModelMetadata, raster::RasterDataType};
+use snafu::{ensure, ResultExt};
+
+use crate::pro::machine_learning::error::{
+    InvalidInputDimensions, InvalidOutputDimensions, MultipleInputsNotSupported, Ort,
+};
+
+use super::MachineLearningError;
+
+// TODO: extract metadata during model import and load it from database here instead of accessing the model file here
```

Comment:
do you still intend to do this?

Reply:
I removed the comment, as it originally targeted loading the model data in the operator, which now does use the database. 07374b850dd9a0d5638233a4a1df5deabb1dceea

We could still think about adding an API handler that autogenerates the MlMetadata, like we do for the OgrDataset. Currently, the user has to specify this manually when adding the model. But right now I think it only saves very little and adds another potential source of failure, if we use it by default.

Reply:
yes i guess it is file like this